en/docs/learn/api-controller/getting-started-with-wso2-api-controller.md

```diff
@@ -594,6 +594,53 @@ For more information, see [Download and Initialize the CTL Tool](#download-and-i
             DefaultApplication Application deleted successfully!
             ``` 
 
+## Change status of an API in an environment
+Follow the instructions below to change the status of an API in an environment using CTL:
```

Comment:
shall we have a note that we need apictl 3.1.1 (or later) version is required?

Reply:
added